src/actions-on-google.ts

```diff
@@ -152,15 +153,15 @@ export class ActionsOnGoogle {
         return client
     }
 
-    set locale(l: string) {
-        if (!SUPPORTED_LOCALES.concat(Object.keys(FALLBACK_LOCALES)).includes(l)) {
+    setLocale(l: string) {
```

Comment:
The method should be `setLocale` function, not a setter. Because we describe `setLocale(...)` about how to specify the locale in the README.md file.

Reply:
Do you think, more generally, that using a setter would be more idiomatic for the language? We can update the README depending on the style? We can also make this a more generalized open question to the community.

Reply:
I think using explicit Java-like setters/getters would be better. Much of the code that accesses private fields used in the code already uses such getters/setters (see send() method in actions-on-google.ts), so it would be better to be consistent with the rest of code. Additionally, [Google JS Guide](https://google.github.io/styleguide/jsguide.html#features-classes-getters-and-setters) recommends using explicit getX/setX methods.

Reply:
I agree the idea of which we adjust the code style to the TypeScript. But, if the `setLocale` is removed, the backward-compatibility will be lost. I will try to add a new style (it means to add the `locale` property) and try to keep the `setLocale`. Also, I will put the `@deprecated` annotation to the `setLocale` function. How about you?

Reply:
@atulep Thank you for the advice!

@Fleker Nick, atulep recommended the `setLocale`. For backward-compatibility, I think we should keep the `setLocale` function, but I intend to follow your decision about whether we add or not the `locale` property to set the locale from users.

Reply:
The library is currently in 'developer preview', so it's entirely possible that we'll make changes that are not backwards compatible as we collect feedback.

Reply:
@Fleker Hmm. Ok, I forget the backward-compatibility. But, at least, I would like to fix some compile errors with this pull request as soon as possible, because currently the master branch is not good condition. For example, could we create the issue of the `locale` property as a new issue? If yes, we can separate the compile error fix and the spec change of the `setLocale`.

Reply:
Yeah it's fine to keep `setLocale` for right now. Fixing some of the syntax issues are higher priority.

Reply:
@Fleker Thank you for your understanding!